docs/docs/reference/other-new-features/export.md

```diff
@@ -6,7 +6,7 @@ movedTo: https://docs.scala-lang.org/scala3/reference/other-new-features/export.
 
 An export clause defines aliases for selected members of an object. Example:
 
-```scala
+```scala sc-name:Model.scala
```

Comment:
What do you think of using `sc-id` instead of name? “id” conveys better the requirement of uniqueness IMHO.

Reply:
There was a version with `sc-id` and I agree with you about the intuitive requirement of uniqueness. We've chosen `sc-name` because we display it on page and we wanted authors to pick more meaningful name than 1, 2, 3...